mobile-widgets/qml/main.qml

```diff
@@ -552,8 +554,16 @@ if you have network connectivity and want to sync your data to cloud storage."),
 		property color contrastAccentColor: "#FF5722" // used for delete button
 		property color lightDrawerColor: "#FFFFFF"
 		property color darkDrawerColor: "#424242"
+		property int initialWidth: rootItem.width
 		Component.onCompleted: {
-			setupUnits()
+			// break the binding
+			initialWidth = initialWidth * 1
```

Comment:
That is very obscure to someone not familiar with QML / JavaScript and might be worthy of a comment.
Is that a way to convert to int? How about `initialWidth + 0`?

Reply:
Actually, the comment is there. This is about breaking the binding in QML. Normally initialWidth would be "bound" to the value it got initialized with as a property - so whenever width changes, initialWidth would change. By making it the result of a computation, we are breaking this binding. I'm sure adding 0 would work the same.
https://doc.qt.io/qt-5/qtqml-syntax-propertybinding.html for more information

Reply:
> Actually, the comment is there. This is about breaking the binding in QML. Normally initialWidth would be "bound" to the value it got initialized with as a property - so whenever width changes, initialWidth would change. By making it the result of a computation, we are breaking this binding. I'm sure adding 0 would work the same.

Whaat? I don't think I want to understand that. :-P